tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "lib": ["es2015"],
+    "strict": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
+    "baseUrl": ".",
+    "paths": {
```

Comment:
these paths could perhaps also be automated btw:

```
var cur = JSON.parse(current)
var next = {...cur, paths: generate()}
var doc = JSON.stringify(next, null, 2) + '\n'
```

Reply:
refactored paths to be pattern based, all lint packages should now be matched.

```json
"paths": {
  "remark-*": ["packages/remark-*"]
}
```